tests/test_loads.py

```diff
@@ -111,19 +112,73 @@ def test_sparse_input(self):
 
     def test_quoted_null(self):
         ARFF_WITH_NULL = '''% XOR Dataset
-@RELATION XOR
-
-@attribute 'bc' {'?','Y'}
-
-@DATA
-'Y'
-'?'
-'Y'
-'?'
-%
-%
-% '''
+        @RELATION XOR
```

Comment:
Is there a reason why you added those indentations?

Reply:
No, just a mistake.